### PR TITLE
WIP: Nativestack

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xnfhqgchwph3h4td00ackwqkjr

[download apk](http://filestorage.bluewallet.io:1488/1592707145.apk)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/8pwb6kt35nvzqyw20wdtwffukw

[download apk](http://filestorage.bluewallet.io:1488/1592793557.apk)

File: ios/BlueWallet-Bridging-Header.h [hunk 1]
```diff
@@ -1,4 +1,4 @@
-//
+  //
```
Comment (Overtorment) [inline ios/BlueWallet-Bridging-Header.h:1]: remove from diff

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/58764y22y7uk063c88rg765jhr

[download apk](http://filestorage.bluewallet.io:1488/1592879947.apk)

**Pull request closed without merging**